packages/upload-api/test/handlers/index.js

```diff
@@ -171,4 +171,78 @@ export const test = {
     assert.ok(receipt.out.error)
     assert.equal(receipt.out.error?.name, 'ShardNotFound')
   },
+  'index/add should publish index claim': async (assert, context) => {
+    const { proof, spaceDid } = await registerSpace(alice, context)
+    const contentCAR = await randomCAR(32)
```

Comment:
nit: should we randomBlob instead since we upload blob later and uniform more towards new world?

Reply:
Hmm, `index/add` only really makes sense if you're storing a CAR though...I can't create an index for a `randomBlob`.